x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/X-2State.java.st

```diff
@@ -7,6 +7,9 @@
 
 package org.elasticsearch.compute.aggregation;
 
+ $if(v2_BytesRef)$
+import org.apache.lucene.util.BytesRef;
+$endif$
```

Comment:
If you do:
```
// begin generated imports
```
then you should be able to import it unconditionally.

It makes the generated code less clean, but madness lies down this path.